src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
Link to a stylesheet must have an `.css` extension, not `.scss`. SCSS files are preprocessed and should be compiled to CSS before being linked in HTML.